### PR TITLE
Generate Broker RedirectURL script for bash shell

### PR DESCRIPTION
**Pull request by brandwe**
Wrote a shell script in bash that can run in Linux or Mac that will generate replyURL for debug or release, and take command line parameters

### CODE REVIEW

**Comment by weijjia** [comment]
@brandwe Can you also update the readme please?

**Comment by brandwe** [comment]
@weijjia Done!

File: brokerRedirectPrint.sh [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+java_bin=$JAVA_HOME/jre/bin
+keytool=$java_bin/keytool
+
+debug_key_store=$HOME/.android/debug.keystore
+release_alias="androiddebugkey"
+release_key_store=$HOME/.android/release.keystore
+release_password="android"
+android_key_store=
+
+#---------------------------------
```
Comment (weijjia) [inline brokerRedirectPrint.sh]: nit: why you need this?

File: brokerRedirectPrint.sh [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  done
+  echo "${encoded}"    # You can either set a return variable (FASTER) 
+  REPLY="${encoded}"   #+or echo the result (EASIER)... or both... :p
+}
+#---------------------------------
+
+#---------------------------------
+# Simple Func() to generate the hash from the certificate provided by the android keystore.
+makeTag()
+{
```
Comment (weijjia) [inline brokerRedirectPrint.sh]: nit: make the parenthesis to the upper line.

File: brokerRedirectPrint.sh [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+   tag=`$keytool -storepass $release_password -exportcert -alias $release_alias -keystore $android_key_store | openssl sha1 -binary |  openssl base64`
+}
+#---------------------------------
+
+#---------------------------------
+makeReplyURL()
+{
```
Comment (weijjia) [inline brokerRedirectPrint.sh]: same as above

File: brokerRedirectPrint.sh [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+   printf "msauth://%s/%s\n" $package_name_encoded $tag
+}
+#---------------------------------
+
+# Simple Func() to do release
+makerelease()
+{
```
Comment (weijjia) [inline brokerRedirectPrint.sh]: same as above

File: brokerRedirectPrint.sh [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    
+    if [ -f "$release_key_store" ]; then
+    
+     android_key_store=$release_key_store
+     
+echo "We are using the following values"
```
Comment (weijjia) [inline brokerRedirectPrint.sh]: indent

**Comment by weijjia** [comment]
Looks good to me. My only comments are the format. Please make sure the coding style is consistent with the repo. :)

**Comment by brandwe** [comment]
@weijjia updated. Feel free to merge and put in to master. It'll help a lot of folks out.

**Comment by weijjia** [comment]
Merging now. :shipit:

**Pull request merged**